README.en.md

```diff
@@ -423,6 +424,10 @@ This is intended to better understand what the problem is if something is not wo
 
 _Note that `wovnCacheDisable` and `wovnDebugMode` is only available when debugMode is turned on in your wovnjava configuration._
 
+### 2.15. encoding
+
+You can choose to optionally explicitly specify the encoding used by your HTML. If `encoding` is not set, wovnjava will attempt to automatically detect the encoding. 
+
```

Comment:
It is easier to understand there are candidates like UTF-8, Shift_JIS, EUC-JP, etc.

Reply:
Let me add a few common examples.

Reply:
Linking to (oracle) java documentation about the standard list of encodings might be helpful. https://docs.oracle.com/javase/8/docs/technotes/guides/intl/encoding.doc.html